test/UnitTests/Controllers/RestaurantControllerUnitTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Moq;
+using AskToniApi.Controllers;
+using AskToniApi.Models;
+
+namespace AskToniApi.UnitTests.Controllers
+{
+    public class RestaurantControllerUnitTests
+    {
+        [Fact]
+        public void Get_Success_ReturnsCorrectResult()
+        {
+            var mockRecommendationRepository = new Mock<IRecommendationRepository>();
+            var restaurantCollection = new List<Restaurant>();

```

Comment:
Can we declare the variable with the type of the object if it's known at compile time instead of var?, (in this case, List<Restaurant>).

Reply:
Yeah, I'm open to either style. Resharper suggests using var though mainly so that people are forced to name stuff better https://resharper.blogspot.ca/2008/03/varification-using-implicitly-typed.html?m=1  

Any preference Kevin?

Reply:
I agree with some of this guy's ideas, and I do think there are some cases where 'var' does improve readability now.
In this case, I would take back what I said and agree that you should use var on this line, because the instantiation explicitly declares what the class is, re declaring the type would just be noisy.
I would argue though that explicit class declaration would be helpful for a case like:
var object = GetStringsFromDatabase();

Unless I open up an IDE and intellisense the function, there's no way for me to know if the return type could be a string array or list of strings.
In this case, it would be nice for example to have:
List<string> object = GetStringsFromDatabase();

Reply:
Yeah, makes sense. Maybe this can be our approach (basically what you noted):
When using 'new' keyword -> use var
When declaring inline to store return value -> use actual type

Reply:
Finally got around to reading this - agree with the above